File: modulo1/segmento2/ejercicio2/ejercicio2.java/Ejercicio2.java
import java.util.Scanner;

public class Ejercicio2
{
	public static void main(String args[])
	{
		Scanner keyboard = new Scanner(System.in);
		
		System.out.print("Ingrese su nombre: ");
		String name = keyboard.next();
		
		System.out.print("Ingrese su apellido: ");
		String lastName = keyboard.next();
	
		System.out.print("Ingrese su DNI: ");
		int dni = keyboard.nextInt();
		
		System.out.print("Ingrese su direccion: ");
		String street= keyboard.next();

		System.out.print("Ingrese su telefono: ");
		int tel = keyboard.nextInt();

		
		System.out.println("Su nombre y apellido es " + name + " " + lastName + " su documento es "+ dni + " vive en la direccion " + street + " y su telefono es " + tel);
	}
}

File: modulo1/segmento2/ejercicio2/ejercicio2.web/ejercicio2.js
function guardar(){
	var nombre = document.getElementById("uno");
	var apellido = document.getElementById("dos");
	var dni = document.getElementById("tres");
	var direccion = document.getElementById("cuatro");
	var telefono = document.getElementById("cinco");
	
	alert("Su nombre y apellido es " + nombre.value.concat(" " + apellido.value + ", su documento es " + dni.value + ", vive en la direccion " + direccion.value + " y su telefono es " + telefono.value ));
}

